app/javascript/components/concept-map/Concept.tsx

```diff
@@ -10,26 +12,31 @@ import {
   Visibility,
 } from './helpers/concept-visibility-handler'
 import { wrapAnimationFrame } from './helpers/animation-helpers'
-import { PureExerciseProgressBar } from './ExerciseProgressBar'
 
 export const Concept = ({
   slug,
   name,
   web_url,
+  tooltip_url,
   handleEnter,
   handleLeave,
   status,
   isActive,
+  isActiveHover,
   exercises = 0,
   exercisesCompleted = 0,
 }: IConcept & {
   handleEnter: MouseEventHandler
   handleLeave: MouseEventHandler
   status: ConceptStatus
   isActive: boolean
+  isActiveHover: boolean
 }): JSX.Element => {
   const [visibility, setVisibility] = useState<Visibility>('hidden')
   const conceptRef = useRef(null)
+  const [hasFocus, setHasFocus] = useState<boolean>(
```

Comment:
This is ambiguous. Usually when an element is focusable and it contains focusable elements, we'd say "focus within", but in this case I think focus is meant to be "focus is on this concept (or within)". Its initial value is "focus is on this concept (but NOT within)".

So:
- either the logic isn't completely sound
- or the naming isn't perfect

It's fine to keep, but add a comment!